packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/CaptureSnapshotConfirmationDialog.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { fireEvent } from '@testing-library/react';
+import * as React from 'react';
+
+import { renderWithTheme } from 'src/utilities/testHelpers';
+
+import { CaptureSnapshotConfirmationDialog } from './CaptureSnapshotConfirmationDialog';
+
+const props = {
+  error: undefined,
+  loading: false,
+  onClose: vi.fn(),
+  onSnapshot: vi.fn(),
+  open: true,
+};
+
+describe('CaptureSnapshotConfirmationDialog', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('should render the dialog', () => {
+    const { getByTestId, getByText } = renderWithTheme(
+      <CaptureSnapshotConfirmationDialog {...props} />
+    );
+
+    const takeSnapshotButton = getByTestId('confirm');
+    const cancelButton = getByText('Cancel');
+    const title = getByText('Take a snapshot?');
+    const body = getByText(
+      'Taking a snapshot will back up your Linode in its current state, overriding your previous snapshot. Are you sure?'
+    );
+
+    expect(takeSnapshotButton).toBeVisible();
+    expect(cancelButton).toBeVisible();
+    expect(title).toBeVisible();
+    expect(body).toBeVisible();
+  });
```

Comment:
```suggestion
  });
  
```
Nit: could we add spaces between each test case in this file? Small change, but helps with readability.